quick_purchase/views/purchase_view.xml

```diff
@@ -7,13 +7,12 @@
         <field name="arch" type="xml">
             <field name="order_line" position="before">
                 <button name="add_product"
-                        string="Add product"
+                        string="Add"
```

Comment:
you're right

Reply:
I think from an ergonomic point of view it's better to have the button next to the command lines. Indeed, the user can immediately realize the utility of the button. If you put it in the head, it will be drowned in several buttons.

Reply:
@legalsylvain  What do you think of @mourad-ehm argument ? It makes sense

Reply:
Well, matter of taste. The "problem" I see is 
- that you loose space
- It's not the in the Odoo UI Spirit. Buttons are very rarely in the form. Generally in the header.

But well, not a blocking point for me.

Reply:
finally I'm ok with mourad here